src/test/resources/_config.properties

```diff
@@ -1,12 +1,23 @@
 selenium_url=http://127.0.0.1:4723/
-capabilities.app=D:\\projects\\Android\\Android.SauceLabs.Mobile.Sample.app.2.7.1.apk
-capabilities.platformName=ANDROID
-capabilities.deviceName=Pixel_5_API_30
-capabilities.udid=emulator-5554
-capabilities.deviceType=phone
-capabilities.automationName=uiautomator2
-capabilities.appWaitActivity=*
 locale=US
 language=en
 implicit_wait=10
-explicit_timeout=20
\ No newline at end of file
+explicit_timeout=20
+#  ANDROID
+# === ANDROID SETTINGS ===
+android.capabilities.deviceName=Pixel_5_API_30
+android.capabilities.udid=emulator-5554
+android.capabilities.automationName=uiautomator2
+android.capabilities.deviceType=phone
+android.capabilities.appWaitActivity=*
+android.capabilities.app=/Users/okolenchenko/Documents/apk/Android.SauceLabs.Mobile.Sample.app.2.7.1.apk
+# === iOS SETTINGS ===
+capabilities.platformName=iOS
+capabilities.deviceName=iPhone 16 Plus
+capabilities.platformVersion=18.5
+capabilities.automationName=XCUITest
+capabilities.deviceType=phone
+capabilities.app=/Users/okolenchenko/Documents/apk/iOS.Simulator.SauceLabs.Mobile.Sample.app.2.7.1.app
+capabilities.autoAcceptAlerts=true
+capabilities.useNewWDA=true
```

Comment:
On real project we don't need local changes here, you can create _config.properties file for remoute repo and  __config.properties for local properties and put it in gitignore